### PR TITLE
community/gexiv2: modernize, fix license

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: community/gexiv2/APKBUILD [hunk 1]
```diff
@@ -2,37 +2,30 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=gexiv2
 pkgver=0.10.8
-pkgrel=0
+pkgrel=1
 pkgdesc="a GObject-based wrapper around the  Exiv2 library"
```
Comment (TBK) [inline community/gexiv2/APKBUILD]:
```suggestion
pkgdesc="a GObject-based wrapper around the Exiv2 library"
```

**Comment by maxice8** [comment]
On Sat, May 25, 2019 at 05:07:16AM -0700, TBK wrote:
> TBK commented on this pull request.
> 
> 
> 
> > @@ -2,37 +2,30 @@
>  # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
>  pkgname=gexiv2
>  pkgver=0.10.8
> -pkgrel=0
> +pkgrel=1
>  pkgdesc="a GObject-based wrapper around the  Exiv2 library"
> 
> ```suggestion
> pkgdesc="a GObject-based wrapper around the Exiv2 library"
> ```
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/8052#pullrequestreview-242000906

Fixed, thanks

**Comment by TBK** [comment]
:( I am sorry, did not spot it the first time around.

0.12.0 is out. It switched to meson and there are some tests.
https://github.com/GNOME/gexiv2/releases/tag/gexiv2-0.12.0
https://gitlab.gnome.org/GNOME/gexiv2/blob/master/NEWS

**Comment by maxice8** [comment]
On Sat, May 25, 2019 at 05:18:55AM -0700, TBK wrote:
> :( I am sorry, did not spot it the first time around.
> 
> 0.12.0 is out. It switched to meson and there are some tests.
> https://github.com/GNOME/gexiv2/releases/tag/gexiv2-0.12.0
> https://gitlab.gnome.org/GNOME/gexiv2/blob/master/NEWS
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/8052#issuecomment-495914097

I can work on it after german class

**Pull request closed without merging**